controllers/syncer/csi_node.go

```diff
@@ -245,7 +256,7 @@ func (s *csiNodeSyncer) getEnvFor(name string) []corev1.EnvVar {
 			},
 			envVarFromField("NODE_ID", "spec.nodeName"),
 			envVarFromField("KUBE_NODE_NAME", "spec.nodeName"),
-		}
+		}...)
```

Comment:
Why is this change needed ?

Reply:
Here I am concating two slices(list), since the number of element of second slice is not known as slice has dynamic length,  so 3 dots i.e `...`  are added for the go compiler to understand this.
You may check [this](#https://www.golangprograms.com/how-to-concatenate-two-or-more-slices-in-golang.html) for reference.